### DISCUSSION TITLE
Digital Signature / Hash

### DISCUSSION THREAD

**Discussion by Nurbinch** [Open-Systems-Pharmacology/Forum#1318] [Questions & Problems]
Dear,

Our IT Security guidelines require that we provide either an installer that is digitally signed before being able to deploy it on a users' workstation. I wanted to have PK-Sim installed for one user, but it seems that it is missing the digital signature. In that case we would still be allowed to install it if we can provide the hash (MD5 or SHA256) from the developers. 

Could I ask you to please share the hash (MD5 or SHA256) of pk-sim-portable-setup.zip available [here](https://github.com/Open-Systems-Pharmacology/PK-Sim/releases/tag/v11.0.150):  (or of both PKSim.BatchTool.exe and PKSim.exe)? 

I suspect that some users are also going to request the whole suite. If you don't mind, could I ask you to please share also the hash of OSPSuite-Full.11.0.139.exe ? 

Thank you very much

**Comment by PavelBal** [comment]
@Yuri05 Would be a good idea to include MD5 hash to downloadable assest in every release.

**Comment by Yuri05** [comment]
@Nurbinch I have added the hashes to the OSP Suite Setup / PK-Sim Setup / MoBi Setup

**Reply by Nurbinch** [reply]
@Yuri05 Thank you very much for the quick action, much appreciated!
